### DISCUSSION TITLE
Absurd Bayesian-like estimates

### DISCUSSION THREAD

**Discussion by Mouad-GM** [mboquien/cigale#56] [Q&A] [answered]
Hi,

I need help with this SED fit I'm trying to do. When using the pcigale.ini (file below) with the photometric data (also below) I get absurdly big and negative bayesian-like estimates of dust luminosity and star formation rates (see results below). The best fit values are normal. I tried changing the dust emission model (between casey2012 and dl2014), but the issue persists.

Thank you in advance

[pcigale.txt](https://github.com/mboquien/cigale/files/11571747/pcigale.txt)
[phototmetry data.txt](https://github.com/mboquien/cigale/files/11571754/phototmetry.data.txt)
[results.txt](https://github.com/mboquien/cigale/files/11571755/results.txt)

**Comment by Mouad-GM** [comment]
After debugging, I found the parameter that breaks this: 'E_BV_lines' needs to stay low or the code starts coming up with negative star forming.

**Reply by mboquien** [reply, marked as answer]
Whe E_BV_lines is extremely high, you can indeed run into purely numerical artifacts. I would not recommend going much beyond 2 magnitudes, which is already very high.
